vcpkg.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "cockatrice",
+  "version-string": "0.0.0-dev",
```

Comment:
Do we want this version?

Reply:
It's not actually used anywhere, just required as part of the json and is the version of the json file itself.